templates/kubernetes/base/deployment.yml

```diff
@@ -57,17 +59,7 @@ spec:
                 secretKeyRef:
                   name: cf-keypair
                   key: private_key
-<% end %>          - name: DATABASE_USERNAME
-            valueFrom:
-                secretKeyRef:
-                  name: <% .Name %>
-                  key: DATABASE_USERNAME
-          - name: DATABASE_PASSWORD
-            valueFrom:
-                secretKeyRef:
-                  name: <% .Name %>
-                  key: DATABASE_PASSWORD
----
+<% end %>---
```

Comment:
Can you use the newline suppression syntax here?
```
<%- end %>
---
```

Reply:
oh wow didn't know that was a thing, thats super convenient